pandas/core/frame.py

```diff
@@ -3533,12 +3533,23 @@ def _combine_match_columns(self, other, func, level=None, fill_value=None):
                                    axes=[left.columns, self.index])
         return self._constructor(new_data)
 
-    def _combine_const(self, other, func, raise_on_error=True):
+    def _combine_const(self, other, func,
+                       raise_on_error=True, fill_value=None):
         if self.empty:
             return self
 
-        new_data = self._data.eval(func=func, other=other,
-                                   raise_on_error=raise_on_error)
+        if fill_value is not None:
+            self_copy = self.copy()
```

Comment:
Thanks, @jreback.

Since the `_combine_const` of all classes are designed without `fill_value` parameter, I worry whether modifying the API is a good idea. At least, Series and some member of sparse need do something to keep consistent. Could you give me some advice?